### DISCUSSION TITLE
Resource monitoring

### DISCUSSION THREAD

**Discussion by rquevedo** [Tecnativa/doodba#443] [Q&A] [answered]
Hi folks, im having trouble with a production instance running doodba 13.0, some times, im assume that is when odoo shop load increase, entire odoo get irresponsive via https. Then, i need to restart odoo instance and evrything get fine. Last time that this happen i run an "htop" command and VPS had full available resources and then run "docker stats" and odoo container seems to had available resources. My question is if there are an oficial way or better practice to monitoring doodba instance. Or any idea of where to look the next time that happen.

Thanks in advance,

**Comment by pedrobaeza** [comment, marked as answer]
Run `docker-compose run --rm -l traefik.enable=false odoo pg_activity` to check blocked DB queries, which can be one of the reasons for the locking.

**Reply by rquevedo** [reply]
Thanks @pedrobaeza something like that is what im looking for.

**Comment by yajo** [comment]
> My question is if there are an oficial way or better practice to monitoring doodba instance

The answer is no.

Just use the monitoring tool that best fits your deployments and systems. There are plenty of them, but that's outside of doodba's scope.

**Reply by rquevedo** [reply]
Thanks @Yajo im know that there are plenty monitoring tools but i allways use it in a odoo instance directly deployed to vps. When i moved to doodba in production server and doodba use docker im getting confused in resource managing area. Im mean, for example odoo has a rule of dumb to compute numbers of workers based on number of cores, how can i compute number of worker to doodba instances if there are two or more in one vps?
